src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -55,7 +55,7 @@ public static class TreeExtensions
         /// Predict a target using a decision tree binary classification model trained with the <see cref="FastTreeBinaryClassificationTrainer"/>.
         /// </summary>
         /// <param name="ctx">The <see cref="BinaryClassificationContext"/>.</param>
-        /// <param name="labelColumn">The labelColumn column.</param>
+        /// <param name="labelColumn">The label column.</param>
```

Comment:
>column [](start = 48, length = 6)

Is it column or column name?

Reply:
let me do that on a separate PR; since it touches a lot more files, and is orthogonal to the purpose of this PR.

---
In reply to: [248854888](https://github.com/dotnet/machinelearning/pull/2170#discussion_r248854888) [](ancestors = 248854888)